Feature/add abstract (#130)
optuna/optunahub-registry: template/README.md

You can find more detailed explanation of the following contents in the tutorial.
Looking at [other packages' implementations](https://github.com/optuna/optunahub-registry/tree/main/package) will also help you.

## Abstract

You can provide an abstract for your package here.

> This section is helpful to advertise your package to potential users.

Reviewer: ditto.
```suggestion
This section will help attract potential users to your package.
```
